### PR TITLE
Test AWS 2 SES

### PR DESCRIPTION
**Pull request by ppalaga**
cc @llowinge

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: integration-tests-aws2/README.adoc [hunk 1]
```diff
@@ -9,13 +17,18 @@ To run the tests against the real AWS API, you need to
   (access key ID and secret access key)
 * Export our preferred
   https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions[AWS region].
+* For SES, you additionally need a https://app.mailslurp.com/[MailSlurp account] and API key
+  (the free dev account should be anough)
```
Comment (aldettinger) [inline integration-tests-aws2/README.adoc:21]: Small typo

File: integration-tests-aws2/aws2-ses/src/test/java/org/apache/camel/quarkus/component/aws2/ses/it/Aws2SesTest.java [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                    .until(
+                            () -> {
+                                final ExtractableResponse<Response> response = RestAssured.given()
+                                        .header("x-api-key", mailSlurpApiKey)
+                                        .get("https://api.mailslurp.com:443/inboxes/" + mailboxId + "/emails") //
+                                        .then()
+                                        .statusCode(200)
+                                        .extract();
+                                final List<String> ids = response.jsonPath().getList("id");
+                                LOG.infof("Expected an indentity verification message from SES; got %s",
```
Comment (aldettinger) [inline integration-tests-aws2/aws2-ses/src/test/java/org/apache/camel/quarkus/component/aws2/ses/it/Aws2SesTest.java:106]: Small typo

**Review by aldettinger — APPROVED** [review]
Welld one, that's quite a test :) The comments helps to follow the logic and :+1: for the cleanup at the end.

File: integration-tests-aws2/README.adoc (hunk 1 above)
Comment (ppalaga) [reply integration-tests-aws2/README.adoc:21]: That's the dialect spoken by Slovaks living in South Moravia ;)

**Comment by ppalaga** [comment]
3e702a1: fixed the typos

**Pull request merged**